src/app/view/components/_common/push-publish-dialog/push-publish-dialog.component.html

```diff
@@ -2,7 +2,7 @@
     modal="modal"
     width="500"
     [dismissableMask]="true"
-    [(visible)]="show"
+    [visible]="true"
```

Comment:
What are all this changes in the push-publish component?

Reply:
I realised It didn't need the show param because of the ```*ngIf="pushPublishIdentifier"```